internal/gengapic/client_init_test.go

```diff
@@ -397,18 +397,18 @@ func TestClientInit(t *testing.T) {
 			serv:      servDeprecated,
 			parameter: proto.String("go-gapic-package=path;mypackage,transport=grpc+rest"),
 			imports: map[pbinfo.ImportSpec]bool{
-				{Path: "bytes"}:                           true,
-				{Path: "context"}:                         true,
-				{Path: "google.golang.org/grpc"}:          true,
-				{Path: "google.golang.org/grpc/metadata"}: true,
-				{Name: "gtransport", Path: "google.golang.org/api/transport/grpc"}: true,
-				{Name: "mypackagepb", Path: "github.com/googleapis/mypackage"}:     true,
-				{Path: "fmt"}: true,
+				{Name: "gtransport", Path: "google.golang.org/api/transport/grpc"}:    true,
```

Comment:
I think gofmt will fix alignment here too

Reply:
Same as above; `gofmt` is fighting me here.